Alignment/CTPPS/plugins/CTPPSFastLocalSimulation.cc

```diff
@@ -3,7 +3,7 @@
 * This is a part of TOTEM offline software.
 * Authors: 
 *  Jan Kašpar (jan.kaspar@gmail.com) 
-*    
+*  Cristian Baldenegro (crisx.baldenegro@gmail.com)
```

Comment:
Yes, I will revert the changes. I modified CTPPSFastLocalSimulation.cc to test the UV Pattern input.